tests/test_ble_client_edge_cases.py

```
@@ -317,3 +360,168 @@ def services(self) -> Any:
+            return None
+
+    ble_client.bleak_client = cast(Any, _Backend())
+
+    with pytest.raises(
+        BLEClient.BLEError,
+        match=re.escape(ERROR_MANAGEMENT_AWAIT_TIMEOUT_INVALID),
+    ):
+        if method_name == "pair":
+            ble_client.pair(confirm=True, await_timeout=cast(Any, invalid_timeout))
+        else:
+            ble_client.unpair(await_timeout=cast(Any, invalid_timeout))
```

Ruff (0.15.4)

[warning] `pytest.raises()` block should contain a single simple statement

(PT012)